src/configuration/routes.md

```diff
@@ -139,17 +157,7 @@ For your master environment, this would function as a catch-all domain.
 
 For development environments, we will also be able to handle this. Here is how:
 
-Let's say we have a project on the EU cluster whose ID is "vmwklxcpbi6zq" and
-we created a branch called "add-theme". The generated apex domain of this
-environment will be `add-theme-vmwklxcpbi6zq.eu.platform.sh`.
-If we have a `http://*.{default}/` route defined, the generated route will
-be `http://*---add-theme-vmwklxcpbi6zq.eu.platform.sh/`. This means you could
-put any subdomain before the triple dashes to reach your application.
-HTTP request to both `http://foo---add-theme-vmwklxcpbi6zq.eu.platform.sh/` and
-`http://bar---add-theme-vmwklxcpbi6zq.eu.platform.sh/` URLs will be routed
-to your application properly. However, request to
-`*---add-theme-vmwklxcpbi6zq.eu.platform.sh` will not be routed since it is not
-a legitimate domain name.
+Let's say we have a project on the EU cluster whose ID is "vmwklxcpbi6zq" and we created a branch called "add-theme". The generated apex domain of this environment will be `add-theme-vmwklxcpbi6zq.eu.platform.sh`. If we have a `http://*.{default}/` route defined, the generated route will be `http://*---add-theme-vmwklxcpbi6zq.eu.platform.sh/`. This means you could put any subdomain before the triple dashes to reach your application. HTTP request to both `http://foo---add-theme-vmwklxcpbi6zq.eu.platform.sh/` and `http://bar---add-theme-vmwklxcpbi6zq.eu.platform.sh/` URLs will be routed to your application properly. However, request to `http://*---add-theme-vmwklxcpbi6zq.eu.platform.sh/` will not be routed since it is not a legitimate domain name.
```

Comment:
It's wrong, anyway. The generated route will now include a random string after `add-theme`...